modules/activity_tracker/main.tf

```diff
@@ -32,6 +32,21 @@ resource "ibm_iam_authorization_policy" "atracker_cloud_logs" {
   description                 = "Permit AT service Sender access to Cloud Logs instance ${each.value.instance_id}"
 }
 
+resource "time_sleep" "wait_for_event_stream_auth_policy" {
```

Comment:
your not using this time_sleep anywhere. It needs to be used in the depends_on for the `ibm_atracker_target.atracker_eventstreams_targets` or else its pointless.

Please also leave a code comment explaining why its used:
`# workaround for https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4478`